src/components/SystemPreferences.vue

```diff
@@ -179,6 +188,26 @@ export default {
         @change="updatedCPU"
       />
     </div>
+    <hr>
+
+    <div class="checkbox">
+      <Checkbox
+        :value="telemetry"
+        label="Allow collection of anonymous statistics to help us improve Rancher Desktop"
+        @input="toggleTelemetry"
+      />
+      <v-popover placement="right">
+        <i class="icon icon-info" />
+        <span slot="popover">
+          Rancher Labs would like to collect a bit of anonymized information<br />
```

Comment:
There doesn't seem to be any reason to have explicit `<br>`s here; HTML can do the layout just fine.

Are we still referring to ourselves as Rancher Labs here, post-acquisition?